cfme/common/vm.py

```diff
@@ -391,6 +391,8 @@ def load_details(self, refresh=False):
         else:
             if refresh:
                 toolbar.refresh()
+                # Workaround while BZ 1390157 is still open
+                sel.click(self.find_quadicon())
```

Comment:
There is a way to get the framework to only do an action if a particular bug is open, we should use this here.